source/1-SDLC-organization/README.md

```diff
@@ -175,6 +176,10 @@ To learn more, check the [official doc](https://docs.aws.amazon.com/cli/latest/u
 
     PS: You can inspect what is going to be deployed by clicking "Details" link of "orgStack.Prepare" action.
 
+1. A Validation Email is sent to your inbox, please click on the confirmation link for the deployment to complete.
```

Comment:
can you add a disclamer as well line 104 about this email coming ?

Reply:
And add a BOLD BIG statement here about the fact that it will block the deployment